src/biblebot/constants/bible.py

```
@@ -4,12 +4,18 @@
     "DEFAULT_TRANSLATION",
+    "EMBEDDED_REFERENCE_PATTERNS",
     "PARTIAL_REFERENCE_PATTERNS",
     "REFERENCE_PATTERNS",
     "SUPPORTED_TRANSLATIONS",
+    "TRIGGER_MODE_ANYWHERE",
+    "TRIGGER_MODE_DIRECT_ONLY",
+    "TRIGGER_MODE_SMART",
     "TRANSLATION_ESV",
     "TRANSLATION_KJV",
+    "VALID_TRIGGER_MODES",
 ]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)